internal/databases/postgres/wal_restore_handler.go

```diff
@@ -46,30 +53,30 @@ func HandleWALRestore(targetPath, sourcePath string, cloudFolder storage.Folder)
 	}
 
 	targetWalDir, err := getWalDirName(targetPath)
-	tracelog.ErrorLogger.FatalfOnError("Failed to get WAL directory name: %v\n", err)
+	tracelog.ErrorLogger.FatalfOnError("Failed to get WAL directory name: %s\n", err)
 	sourceWalDir, err := getWalDirName(sourcePath)
-	tracelog.ErrorLogger.FatalfOnError("Failed to get WAL directory name: %v\n", err)
+	tracelog.ErrorLogger.FatalfOnError("Failed to get WAL directory name: %s\n", err)
 
 	tgtHistoryRecords, err := getLocalTimelineHistoryRecords(targetPgData.GetCurrentTimeline(), targetWalDir)
-	tracelog.ErrorLogger.FatalfOnError("Failed to get history data on target cluster: %v\n", err)
+	tracelog.ErrorLogger.FatalfOnError("Failed to get history data on target cluster: %s\n", err)
 	srcHistoryRecords, err := getLocalTimelineHistoryRecords(sourcePgData.GetCurrentTimeline(), sourceWalDir)
```

Comment:
I am curious how does it work in the case of remote `targetWalDir`?

Reply:
My bad. I was too hasty. Will be fixed.